src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -144,6 +146,25 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
         }
 
         this.profileImportFilePath = files[0];
+
+        if (this.profileMods.known.length === 0 || this.profileMods.unknown.length > 0) {
```

Comment:
Nitpick: I think we're only interested in the number of unknown mods here? Checking number of known mods won't probably hurt, but at the same time refreshing the mod list can only solve issues with unknown mods.